static_src/actions/space_actions.js

```diff
@@ -5,27 +5,41 @@
  */
 
 import AppDispatcher from '../dispatcher.js';
+import cfApi from '../util/cf_api';
 import { spaceActionTypes } from '../constants.js';
+import SpaceStore from '../stores/space_store';
 
 export default {
   fetch(spaceGuid) {
     AppDispatcher.handleViewAction({
       type: spaceActionTypes.SPACE_FETCH,
       spaceGuid
     });
+
+    // TODO error action should also be specified here
+    return cfApi.fetchSpace(spaceGuid).then(this.receivedSpace);
```

Comment:
My opinion is that the fetch action should also handle the receive action as the promise resolves. The alternative would be that the UI component calling the action would have to call the receive which feels too boilerplate to me. It's also unnecessary, why would a caller want to fetch and not call the receive? Rather than have purely dumb actions, I think this fetch/receive pattern works well.